docs/root/version_history/v1.16.0.rst

```diff
@@ -159,7 +159,7 @@ New Features
 * watchdog: support randomizing the watchdog's kill timeout to prevent synchronized kills via a maximium jitter parameter :ref:`max_kill_timeout_jitter<envoy_v3_api_field_config.bootstrap.v3.Watchdog.max_kill_timeout_jitter>`.
 * watchdog: supports an extension point where actions can be registered to fire on watchdog events such as miss, megamiss, kill and multikill. See :ref:`watchdog actions<envoy_v3_api_field_config.bootstrap.v3.Watchdog.actions>`.
 * watchdog: watchdog action extension that does cpu profiling. See :ref:`Profile Action <envoy_v3_api_file_envoy/extensions/watchdog/profile_action/v3alpha/profile_action.proto>`.
-* watchdog: watchdog action extension that sends SIGABRT to the stuck thread to terminate the process. See :ref:`Abort Action <envoy_v3_api_file_envoy/extensions/watchdog/abort_action/v3alpha/abort_action.proto>`.
+* watchdog: watchdog action extension that sends SIGABRT to the stuck thread to terminate the process. See :ref:`Abort Action <envoy_v3_api_msg_watchdog.v3alpha.AbortActionConfig>`.
```

Comment:
@envoyproxy/api-shepherds 

Actually, 1.16 still uses the extensions version of this proto so there is a backwards incompatibility issue here.  I think you should rollback the changes to the 1.16 release notes.  Should get advice from api sheperds about how to best proceed here.  I think that the backwards compatibility break should be ok since the abort action shouldn't be widely used.

Reply:
Ah. Curious what api sheperds suggest as well. I believe my change of moving this extension into core breaks the link used in the version history for 1.16.0 causing the ci to fail.

Here's results of running it locally:
```
/source/generated/rst/version_history/v1.16.0.rst:160: WARNING: undefined label: envoy_v3_api_file_envoy/extensions/watchdog/abort_action/v3alpha/abort_action.proto (if the link has no caption the label must precede a section header)
generating indices...  genindex http-routingtabledone
writing additional pages...  searchdone
copying images... [100%] _static/docker_compose_front_proxy.svg
copying static files... ... done
copying extra files... done
dumping search index in English (code: en)... done
dumping object inventory... done
build finished with problems, 1 warning.
Copying tabs assets
Writing redirects...
(good) intro/arch_overview/http/websocket.html --> intro/arch_overview/http/upgrades.html
(good) configuration/observability/access_log.html --> intro/arch_overview/observability/access_logging.html
(good) install/building.html --> start/building.html
(good) install/ref_configs.html --> start/install/ref_configs.html
(good) install/sandboxes/local_docker_build.html --> start/install/sandboxes/local_docker_build.html
(good) install/tools/config_load_check_tool.html --> start/install/tools/config_load_check_tool.html
(good) install/tools/route_table_check_tool.html --> start/install/tools/route_table_check_tool.html
(good) install/tools/schema_validator_check_tool.html --> start/install/tools/schema_validator_check_tool.html
(good) install/tools/tools.html --> start/install/tools/tools.html
(good) install/install.html --> start/start.html
[09:13:48] kbaichoo:envoy:1 $
```

Notice the `1` there is my `PS1` showing the error code from running the command.

Reply:
This is v3alpha, so you are free to make breaking changes like this.